understanding/focus-appearance.html

```diff
@@ -445,498 +372,379 @@ <h4>Inline links</h4>
                   
                   
-                  <p>Nonetheless, consistent results from different testers were obtained for this Success
-                     Criterion by using the focus indicator itself as the gauge of what constitutes the
-                     component being interacted with. For complex components, the three typical focus indicators
-                     are as follows:
+                  <p>それにもかかわらず、この達成基準では、フォーカスインジケータそのものを、操作されるコンポーネントを構成するものの基準として使用することで、様々なテスターから一貫した結果が得られた。複雑なコンポーネントの場合、典型的なフォーカス指標は次の三つである:
                   </p>
                   
                   
                   <ul>
                      
-                     <li>Focus indicator around only the whole component</li>
+                     <li>コンポーネント全体を囲むフォーカスインジケータ</li>
                      
-                     <li>Focus indicators around both the component and subcomponent</li>
+                     <li>コンポーネントとサブコンポーネントの両方を囲むフォーカスインジケータ</li>
                      
-                     <li>Focus indicator around only the subcomponent</li>
+                     <li>サブコンポーネントのみを囲むフォーカスインジケータ</li>
                      
                   </ul>
                   
                   
-                  <p>Each of these will be discussed, using a tablist as a familiar complex component.</p>
+                  <p>これらのそれぞれについて、よく知られている複合的なコンポーネントとしてタブリストを用いて説明する。</p>
                   
                   
-                  <h3>Focus indicator around only the whole component</h3>
+                  <h3>コンポーネント全体を囲むフォーカスインジケータ</h3>
                   
                   <figure id="component-complex-whole">
-                     <img src="img/component-complex-whole.png" alt="Three tab buttons shown with a dark blue rectangle around all three tab buttons." />
+                     <img src="img/component-complex-whole.png" alt="三つのタブボタンの全てを囲む濃い青色の矩形が表示されている。" />
                      
-                     <figcaption>Figure 20  A tablist with a focus indicator around only the whole.</figcaption>
+                     <figcaption>図 20 全体だけにフォーカスインジケータを表示したタブリスト。</figcaption>
                      
                   </figure>    
                   
                   
-                  <p>When the focus indicator is shown only around the whole tablist, the user is guided
-                     to considering the tablist as a single user component. The tab items within it are
-                     visually distinguished between selected and unselected states (and visual indicators
-                     of selection state must meet the criteria given in <a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">1.4.11 Non-text Contrast</a>).
+                  <p>タブリスト全体にだけフォーカスインジケータが表示されている場合、利用差はタブリストを一つのユーザコンポーネントとして考えるように誘導される。その中のタブ項目は、選択状態と非選択状態が視覚的に区別される (そして選択状態の視覚的インジケータは、<a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">1.4.11 非テキストのコントラスト</a>で示された基準を満たなければならない)。
```

Comment:
TYPO あったので修正します。(利用差 -> 利用者)
「ユーザコンポーネント」は本当は「ユーザインタフェースコンポーネント」なのでしょうが、原文でも user component になっているので、まあ、致し方なしですね...

```suggestion
                  <p>タブリスト全体にだけフォーカスインジケータが表示されている場合、利用者はタブリストを一つのユーザコンポーネントとして考えるように誘導される。その中のタブ項目は、選択状態と非選択状態が視覚的に区別される (そして選択状態の視覚的インジケータは、<a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">1.4.11 非テキストのコントラスト</a>で示された基準を満たなければならない)。
```